build_depends.repos

```diff
@@ -7,7 +7,3 @@ repositories:
     type: git
     url: https://github.com/HarvestX/h6x_tf_manager.git
     version: humble
-  h6x_tf_manager:
-    type: git
-    url: git@github.com:HarvestX/h6x_tf_manager.git
```

Comment:
We want https url here

Reply:
https url exists already.
I removed ssh of tf_manager because both ssh and https existed in the file before the change.

Reply:
Oh sorry, I didn't directly check the file.
Thank you so much!